lib/helpers/mail/model/EmailAddress.php

```diff
@@ -0,0 +1,51 @@
+<?php namespace SendGrid\Helpers\Mail\Model;
+
+class EmailAddress implements \JsonSerializable
+{
+    public $name;
+    public $email;
+
+    public function __construct($name = null, $emailAddress)
+    {
```

Comment:
Doing this will force the developer to assign `null` everytime this object is needed, default values should always be in the end, implementing this way:

```php
(new EmailAddress(null, 'email@test.com')
(new EmailAddress(null, 'other@email.com')
```

If you don't want to assign a name, it should look like this

```php
(new EmailAddress('email@address.com')
(new EmailAddress('other@address.com', 'name')
```